src/common/components/templates/InfoBanner.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { FaTimes } from "react-icons/fa";
+import * as Toast from "@radix-ui/react-toast";
+
+export default function InfoToast() {
+  const [isDisplayed, setIsDisplayed] = useState(false);
+  const [message, setMessage] = useState("");
+  const [duration, setDuration] = useState(10000);
+  const router = useRouter();
+  const { pathname, query } = router;
+  const userFarcasterName = query.handle;
+
+  useEffect(() => {
+    let storedState;
+
+    if (pathname === "/homebase") {
+      storedState = localStorage.getItem("homebaseToastDisplayed");
+      if (!storedState && localStorage.getItem("privy:token") !== null) {
+        setIsDisplayed(true);
+        setMessage(
+          "Your homebase is a space that only you can see. Click the paintbrush to customize it 🚀",
+        );
+        setDuration(10000);
+      } else {
+        setIsDisplayed(false);
+      }
+    } else if (
+      pathname.startsWith("/s/") &&
+      userFarcasterName === "farcaster"
```

Comment:
why do we have this check? I think you wanted to check if the user can modify the space, not check the handle as an explicit string

Reply:
I think that was a test, gonna check

Reply:
Just checked and what I want to do here is to check if he can modify a.k.a the connected user = the owner of the space. in my local env I am always farcaster for somereason, so that was the only way to test it. Good Catch